addons/sms/tools/sms_api.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import werkzeug.urls
+
+from odoo import _
+from odoo.addons.iap.tools import iap_tools
+
+DEFAULT_ENDPOINT = 'https://iap-sms.odoo.com'
```

Comment:
Could you change this to the new format of the route while you're at it?
```suggestion
DEFAULT_ENDPOINT = 'https://sms.api.odoo.com'
```